e2e/fiat_tf_test.go

```diff
@@ -143,42 +160,67 @@ func TestFiatTFConfigureMinterController(t *testing.T) {
 	}
 	require.Equal(t, expectedShowMinterController.MinterController, showMCRes.MinterController)
 
+	// ACTION: Configure Minter Controller while TF is paused
+	// EXPECTED: Success; Minter Controller is configured with Minter
+	// Status:
+	// 	minterController1 -> minter1
+
+	w = interchaintest.GetAndFundTestUsers(t, ctx, "default", math.OneInt(), noble, noble)
+	minterController2 := w[0]
+	minter2 := w[1]
+
+	pauseFiatTF(t, ctx, val, nw.fiatTfRoles.Pauser)
+
+	_, err = val.ExecTx(ctx, nw.fiatTfRoles.MasterMinter.KeyName(), "fiat-tokenfactory", "configure-minter-controller", minterController2.FormattedAddress(), minter2.FormattedAddress())
+	require.NoError(t, err, "error configuring minter controller")
+
+	showMCRes, err = showMinterController(ctx, val, minterController2)
+	require.NoError(t, err, "failed to query show-minter-controller")
+	expectedShowMinterController = fiattokenfactorytypes.QueryGetMinterControllerResponse{
+		MinterController: fiattokenfactorytypes.MinterController{
+			Minter:     minter2.FormattedAddress(),
+			Controller: minterController2.FormattedAddress(),
+		},
+	}
+	require.Equal(t, expectedShowMinterController.MinterController, showMCRes.MinterController)
+
 	unpauseFiatTF(t, ctx, val, nw.fiatTfRoles.Pauser)
 
 	// ACTION: Configure Minter Controller from non Master Minter account
 	// EXPECTED: Request fails; Minter Controller not configured with Minter
 	// Status:
 	// 	minterController1 -> minter1
+	// 	minterController2 -> minter2
 
 	w = interchaintest.GetAndFundTestUsers(t, ctx, "default", math.OneInt(), noble, noble, noble)
-	minterController2 := w[0]
-	minter2 := w[1]
+	minterController3 := w[0]
+	minter3 := w[1]
 	alice := w[2]
 
-	_, err = val.ExecTx(ctx, alice.KeyName(), "fiat-tokenfactory", "configure-minter-controller", minterController2.FormattedAddress(), minter2.FormattedAddress())
+	_, err = val.ExecTx(ctx, alice.KeyName(), "fiat-tokenfactory", "configure-minter-controller", minterController3.FormattedAddress(), minter3.FormattedAddress())
 	require.ErrorContains(t, err, "you are not the master minter: unauthorized")
 
-	_, err = showMinterController(ctx, val, minterController2)
+	_, err = showMinterController(ctx, val, minterController3)
 	require.Error(t, err, "successfully queried for the minter controller when it should have failed")
 
 	// ACTION: Configure a blacklisted Minter Controller and Minter from blacklisted Master Minter account
 	// EXPECTED: Success; Minter Controller is configured with Minter
 	// Status:
-	// 	minterController1 -> minter1
+	// 	minterController2 -> minter2
```

Comment:
nit: this should have both pairs of minterController/minter, right? (1 & 2)

Reply:
Good eye.
[`f743ec2` (#368)](https://github.com/noble-assets/noble/pull/368/commits/f743ec214600f01360b2aa9befa57c4197a5010e)